src/demo/pages/TooltipPage/examples/Tooltip.Basic.Example.tsx

```diff
@@ -0,0 +1,53 @@
+/* tslint:disable:no-unused-variable */
+import * as React from 'react';
+/* tslint:enable:no-unused-variable */
+
+import {
+  Tooltip,
+  Button
+} from '../../../../index';
+
+export interface ITooltipBasicExampleState {
+  isTooltipVisible?: boolean;
+}
+
+export class TooltipBasicExample extends React.Component<any, ITooltipBasicExampleState> {
+  private _menuButtonElement: HTMLElement;
+
+  public constructor() {
+    super();
+
+    this._onDismiss = this._onDismiss.bind(this);
+
+    this.state = {
+      isTooltipVisible: false,
+    };
+  }
+
+  public render() {
+    let { isTooltipVisible } = this.state;
+
+    return (
+      <div className='ms-TooltipExample'>
```

Comment:
This example demos rendering the component but doesn't really demo proper usage. We probably need to break that down. Example; I have a button, and I want a custom tooltip on it. That is, when I hover over it, for some period of time, it shows up. How do i do that? Do I really need to manage any state?

```ts
<Tooltip content={
  <span>I'm <b>fancy</b> tooltip content.</span>
}>
<Button>Hello world</Button>
</Tooltip>
```

Now when i hover over the button it shows a fancy tooltip.

Other questions:

* How do i control hover delay?
* How long does it stay after I move the mouse away?
* Does it stay if i'm hovering over it?
* Does it dismiss other tooltips if I hover over other things?